src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -254,15 +254,28 @@ namespace ts.codefix {
         }
     }
 
-    function mayDeleteParameter(checker: TypeChecker, sourceFile: SourceFile, parameter: ParameterDeclaration, isFixAll: boolean): boolean {
+    function mayDeleteParameter(checker: TypeChecker, sourceFile: SourceFile, parameter: ParameterDeclaration, sourceFiles: readonly SourceFile[], program: Program, cancellationToken: CancellationToken, isFixAll: boolean): boolean {
         const { parent } = parameter;
         switch (parent.kind) {
             case SyntaxKind.MethodDeclaration:
                 // Don't remove a parameter if this overrides something.
                 const symbol = checker.getSymbolAtLocation(parent.name)!;
-                if (isMemberSymbolInBaseType(symbol, checker)) return false;
-                // falls through
+                return !getPropertySymbolsFromBaseTypes(symbol.parent!, symbol.name, checker, _ => true);

```

Comment:
Does the same problem not exist for method parameters?

Reply:
Seems like it does: https://www.staging-typescript.org/play?ts=4.1.0-beta&ssl=2&ssc=9&pln=2&pc=18#code/MYGwhgzhAEBCkFNoG8BQ0PQGYHscAoAPALmgDsBXAWwCMEAnAShQF9oB6d6AJQSpwBuSCpQgIAJtAAOYemCoIALg1QtUqUJBgARBgEshkhIWVlxMeGJTpMuAiXLU6Ta5jfQIFKQwB0doowA3DYYaixAA